core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1391,8 +1391,9 @@ class SparkContext(config: SparkConf) extends Logging {
    * standard mutable collections. So you can use this with mutable Map, Set, etc.
    */
   @deprecated("use AccumulatorV2", "2.0.0")
-  def accumulableCollection[R <% Growable[T] with TraversableOnce[T] with Serializable: ClassTag, T]
-      (initialValue: R): Accumulable[R, T] = {
+  def accumulableCollection[R : ClassTag, T](initialValue: R)
```

Comment:
This and the `SequenceFileRDDFunctions` change are the only questionable ones. This should be the equivalent declaration that context bounds desugar into, but the question is whether this actually induces an obscure API change.